src/cargo/ops/cargo_output_metadata.rs

```diff
@@ -65,6 +72,8 @@ pub struct ExportInfo {
     version: u32,
     workspace_root: PathBuf,
     metadata: Option<toml::Value>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    workspace_profiles: Option<TomlProfiles>,
```

Comment:
> The output only changes if the user has included any specific profile settings in their Cargo.toml file, it doesn't expose default profiles that have not been modified. 

For me, the question is what the intended use case is we are covering.  All callers will need to reinvent cargo's profile system to fully interpret this, including reading from config.  Is that ok?  Can we change this in the future if we find it doesn't work?

Reply:
One specific use case doesn't tell us much though.  We are looking at adding a new feature and we need to consider other users and future users to make sure we are providing the right thing.

Do we understand enough to move forward either direction?

Reply:
In my usecase I would like to have the default profiles available. My usecase would be to provide CMake users an option to select one of the available cargo profiles. For this I would like to have
a) name of the profile
b) path of the profile (mainly because `dev` maps to `debug`)

This list should include the default profiles, although I'm not sure about `test` and `bench` though, since those profiles are weird and artifacts don't land at a predictable location due to the hash being added.